### DISCUSSION TITLE
求助~建mysql外部表时，支持一个doris表对多个mysql表吗，业务系统有分表，分了很多张表。

### DISCUSSION THREAD

**Discussion by denganming** [apache/doris#10744] [A - General / Q&A]
`CREATE EXTERNAL TABLE example_db.table_mysql
(
	k1 DATE,
	k2 INT,
	k3 SMALLINT,
	k4 VARCHAR(2048),
	k5 DATETIME
)
ENGINE=mysql
PROPERTIES
(
	"host" = "127.0.0.1",
	"port" = "8239",
	"user" = "mysql_user",
	"password" = "mysql_passwd",
	"database" = "mysql_db_test",
 "table" = "mysql_table_test",
 "charset" = "utf8mb4"
)`

**Comment by dataalive** [comment]
目前通过单一的外表没办法直接帮你屏蔽分库分表。但是可以用Flink CDC + Doris Connector先聚合后再灌入Doris。

**Comment by hf200012** [comment]
> `CREATE EXTERNAL TABLE example_db.table_mysql ( k1 DATE, k2 INT, k3 SMALLINT, k4 VARCHAR(2048), k5 DATETIME ) ENGINE=mysql PROPERTIES ( "host" = "127.0.0.1", "port" = "8239", "user" = "mysql_user", "password" = "mysql_passwd", "database" = "mysql_db_test", "table" = "mysql_table_test", "charset" = "utf8mb4" )`

暂时不支持odbc外表方式对应mysql数据库里分库分表，只能创建多个odbc外表，建议你使用Flink CDC + Doris Flink Connector方式将Mysql分库分表的数据接入到Doris里
